src/load.test.ts

```diff
@@ -92,7 +92,10 @@ describe('load behaves as expected', () => {
   it('reads the config based on process.env.RUNTIME_ENVIRONMENT', () => {
     load()
 
-    expect(mockedReadConfigFile).toHaveBeenCalledWith(RUNTIME_ENVIRONMENT)
+    expect(mockedReadConfigFile).toHaveBeenCalledWith(
+      expect.any(String),
```

Comment:
Why the extra `expect` here? Maybe this is some jest magic I'm not familiar with but the normal signature of  `toHaveBeenCalledWith` is `toHaveBeenCalledWith(arg1, arg2, ...)`.

so if you expect `mockedReadConfigFiles` to be called with two args, why not just `toHaveBeenCalledWith(basePath, RUNTIMIE_ENVIRONMENT)`?

Reply:
it's two different kind of `expect`s. The second one is a helper that is used a placeholder to be happy with 'any kind of string'.

The idea is to `expect` on things that a particular test is validating. Here, it's checking if the `RUNTIME_ENVIRONMENT` is passed correctly. Thus, `mockedReadConfigFiles` must be called with (<any string we don't care about in this test>, RUNTIME_ENVIRONMENT)`

Reply:
got it, ok